src/transformers/models/mctct/modeling_mctct.py

```diff
@@ -566,13 +566,13 @@ def __init__(self, config: MCTCTConfig):
 
     def forward(
         self,
-        input_features,
-        attention_mask,
-        head_mask,
-        output_attentions=False,
-        output_hidden_states=False,
-        return_dict=True,
-    ):
+        input_features: torch.Tensor,
+        attention_mask: Optional[torch.Tensor] = None,
+        head_mask: Optional[torch.Tensor] = None,
```

Comment:
Careful, these weren't optional before!

Reply:
I see - I've removed the `Optional` parts for these variables now